data/json/monsters/insect_spider.json

```diff
@@ -1788,5 +1788,36 @@
     "death_function": [ "NORMAL" ],
     "special_attacks": [ { "type": "leap", "cooldown": 4, "max_range": 4, "allow_no_target": true }, [ "EAT_CROP", 120 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "LARVA", "STUMBLES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_stag_beetle",
+    "type": "MONSTER",
+    "name": { "str": "Huge Stag Beetle" },
+    "description": "Once a priceless bug, it has grown to the size of a calf.  Although you doubt that it's going to attack you, those large pincers could probably break your leg.",
```

Comment:
Priceless?

Reply:
Could you explain in a bit more detail what you're trying to ask? I'm a bit tired

Reply:
"Once a priceless bug" in the description? (Understand on tired!)

Reply:
Once a priceless bug refers to the fact the stag beetle used to be highly sought after pre-cdda for various reasons. One of which was their ability to help keep gardens healthy. Or just because collectors are collectors. 
Tl;DR: It's mainly there for fluff

Reply:
I partially did the laugh emoji because - re TL;DR - I practically never will **_not_** have read the entire posting, BTW.

Reply:
All good lol